### ISSUE TITLE
Implement taunts

### ISSUE THREAD

**Issue by coffenbacher** [SFTtech/openage#657]
Critical feature we need 30 - wololo and 10 - "I'll beat you back to Age of Empires" to go along with cheats. Depends on #109 

14 14 14 14 14 14 14 14

**Comment by MichaHoffmann** [comment]
Is this issue still open? If so could i work on it?

**Comment by TheJJ** [comment]
Yes, it's still open. The question is how you want to implement this: We have no chat system yet, just playing sounds is functional. We also need to think about how the sounds are stored/registered:
* We can do it like the original game: just dump files in a folder and prefix them with the id.
  * Pro: Simple
  * Con: Modpacks can't add same-id files then (-> it'll be impossible to add taunts from a mod manager)
* We can use nyan: Sounds are registered through nyan, modpacks can specify "preferred" ids. We'd need a GUI where ids can be customized
  * Pro: Extensible and customizable
  * Con: Way more complicated

Because we have to think of this first, I "downgraded" the issue from "just do it" to "proposal".

**Comment by MichaHoffmann** [comment]
It sounds like one should use nyan ( i have no idea what that is! ). Im looking for a good Issue to get started on this project, do you have any recommendations? ( How about #109 ? )

**Comment by TheJJ** [comment]
@MichaHoffmann i suggest that you come by in our chatroom, more interactive discussion is possible there. Generally, we're currently working on improving the engine foundation greatly which kind of blocks issues like this.
